src/HOOKs/useClickOutside/useClickOutside.tsx

```diff
@@ -0,0 +1,26 @@
+import * as React from "react";
+
+export function useClickOutside(
+	ref: React.RefObject<HTMLElement>,
+	handler: (e: MouseEvent) => void
+) {
+	React.useEffect(
+		() => {
+			const listner = (event: MouseEvent) => {
```

Comment:
`event` есть в глобальной области видимости и тут мы его переопределяем для этой функции, так что лучше просто `e`
```suggestion
			const listener = (e: MouseEvent) => {
```